diff --git a/test/e2e/app-dir/app-prefetch/prefetching.stale-times.test.ts b/test/e2e/app-dir/app-prefetch/prefetching.stale-times.test.ts
index 3b4c3b14..907f0b63 100644
--- a/test/e2e/app-dir/app-prefetch/prefetching.stale-times.test.ts
+++ b/test/e2e/app-dir/app-prefetch/prefetching.stale-times.test.ts
@@ -128,7 +128,7 @@ describe('app dir - prefetching (custom staleTime)', () => {
   })
 
   // FIXME: Flaky test - investigate and re-enable
-  it.skip('should not re-fetch cached data when navigating back to a route group', async () => {
+  it('should not re-fetch cached data when navigating back to a route group', async () => {
     let act: ReturnType<typeof createRouterAct>
     // Just installing so that the page doesn't automatically move past dynamic stale time
     createTimeController()
